docs/dev/check_references.md

```diff
@@ -99,6 +111,76 @@ The complete list of mandatory and optional attributes for the `manifest.json` f
+| `legal_terms[eula] `                                 | String                                                                  | Mandatory                                  | Relative path to the EULA (End User License Agreement) PDF in relation to this manifest.                                                                                                                                                                   |   |
+| `manifest_version`                                   | String Enum                                                             | Mandatory    | Version of the manifest schema. Supported values include `1.0.0` and `2.0.0`.                                                                                                                                                                                     |   |
+| `pricing`                                            | Array of Object                                                         | Mandatory                                  | List of objects representing the pricing model of the integration. See [Marketplace GitHub repository][1] for pricing details. The Marketplace GitHub repository is private, email marketplace@datadog.com for access.                                                                                                                                                                                          |   || `tile`                                               | Object                                                                  | Mandatory                                  | Information about this offering                                                                                                                                                                                                                                |   |
+| `tile[media]`                                        | Array of Object                                                         | Mandatory, can be `[ ]`                    | Information about various image and video style objects that are presented in the media gallery carousel on the listing page.                                                                                                                             |   |
+| `tile[media[media_type]]`                            | String or Enum                                                          | Mandatory                                  | The type of media this is. If you're setting this field as an Enum, only `image` and `video` are accepted.                                                                                                               |   |
```

Comment:
We've implemented this as an enum, so to a user the only allowed values are `video` and `image`. Suggestion to update the wording to be a bit clearer there:

```suggestion
| `tile[media[media_type]]`                            | String or Enum                                                          | Mandatory                                  | The type of media this is. Allowed values are `image` and `video`.                                                                                                               |   |
```